chat_logger/admin.py

```diff
@@ -0,0 +1,43 @@
+from django.contrib import admin
+from . import models
+
+
+class Log5XXErrorAdmin(admin.ModelAdmin):
```

Comment:
This model admin class will show the data (errors) from the database models in the admin panel in a manageable and informative manner.